### PR TITLE
added delete vocab route to unification

### PR DESCRIPTION
**Pull request by SchrodingersCat00**

### CODE REVIEW

File: services/content-unification/unification.py [hunk 1]
```diff
@@ -137,3 +141,255 @@ def delete_from_graph(subjects, graph):
...
+def remove_vocab_source_datasets(vocab_uuid: str, graph: str):
+    query_template = Template("""
+PREFIX void: <http://rdfs.org/ns/void#>
+PREFIX dct: <http://purl.org/dc/terms/>
+PREFIX ext: <http://mu.semte.ch/vocabularies/ext/>
+PREFIX mu: <http://mu.semte.ch/vocabularies/core/>
+
+WITH $graph
+DELETE {
+    ?sourceDataset ?sourceDatasetPred ?sourceDatasetObj
```
Comment (MikiDi) [inline services/content-unification/unification.py:210]: Om consequent te zijn zou ik net zoals in de query  hierboven ook `?vocab ext:sourceDataset ?sourceDataset .` meenemen bij het verwijderen.

File: services/content-unification/unification.py [hunk 2]
```diff
@@ -137,3 +141,255 @@ def delete_from_graph(subjects, graph):
...
+WITH $graph
+
+DELETE {
+    ?classPart ?classPartPred ?classPartObj .
+}
+WHERE {
+    ?vocab a ext:VocabularyMeta ;
+           mu:uuid $vocab_uuid .
+    ?vocab ext:sourceDataset ?sourceDataset .
+    ?sourceDataset void:classPartition ?classPart .
```
Comment (MikiDi) [inline services/content-unification/unification.py:324]: Deze query en degene hieronder kunnen misschien makkelijk gecombineerd worden tot 1 door de predicaten `void:classPartition` en `void:propertyPartition` deel te binden via een `VALUES` blok?

**Review by MikiDi — CHANGES_REQUESTED** [review]
Haal je ook nog prefixes weg waar ze niet nodig zijn?

**Comment by Milan-VLIZ** [comment]
I suppose this one is good to merge? Or no longer relevant?

**Comment by SchrodingersCat00** [comment]
This can and should be merged, this is needed to clean up the downloaded files and other relevant metadata that is created when a vocabulary is added.

**Comment by SchrodingersCat00** [comment]
@MikiDi or @Jan-PieterBaert does this need to be expanded for LDES vocabs?

**Comment by Jan-PieterBaert** [comment]
> @MikiDi or @Jan-PieterBaert does this need to be expanded for LDES vocabs?

@SchrodingersCat00 for LDES the docker container that the `ldes-consumer-manager` created should be destroyed

**Comment by SchrodingersCat00** [comment]
Is this something you could implement easily? I have no experience controlling docker via scripting whatsoever

**Comment by Jan-PieterBaert** [comment]
the consumer manager can list the containers, using the SDK function to kill a container (see https://docker-py.readthedocs.io/en/stable/containers.html#docker.models.containers.Container.kill ) it should be easy to implement

**Pull request merged**